rust/src/lib.rs

```diff
@@ -1182,17 +1167,18 @@ impl TransactionWitnessSet {
 
 #[wasm_bindgen]
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
-pub struct MsigPubkey {
+pub struct ScriptPubkey {
```

Comment:
I left these names as the new changes ones even though they're basically the same thing since
1) I don't think anyone is using any of the multisig things anyway
2) it's now much more general than multisig so it's less confusing this way